test/core.jl

```diff
@@ -23,9 +23,8 @@
     end
 
     @testset "Hybrid NB" begin
-
         # a test to check that HybridNB successfully replaces KernelNB
-        m1 = HybridNB(y, 2)
+        m1 = HybridNB(y, ["c1", "c2"])
```

Comment:
I am not very experienced in designing packages so I am happy to learn from your experience. Your suggestions are more than welcome.